bindings/wasm/examples/editor.js

```diff
@@ -280,8 +285,10 @@ var Module = {
       const output = await worker.getEmitOutput(editor.getModel().uri.toString());
       const content = output.outputFiles[0].text + 'push2MV(result);';
       const exposedFunctions = [
-        'cube', 'cylinder', 'sphere', 'extrude', 'revolve',
-        'union', 'difference', 'intersection',
+        'cube', 'cylinder', 'sphere', 'tetrahedron', 'extrude', 'revolve',
+        'union', 'difference', 'intersection', 'compose', 'levelSet',
+        'setMinCircularAngle', 'setMinCircularEdgeLength', 'setCircularSegments',
+        'getCircularSegments'
```

Comment:
We could make some separate lists: `constructors`, `staticDefaults`, `memberFunctions` and concatenate them here so we don't have to remember to update them in multiple places.

Reply:
yes, will do